components/pd_client/src/client_v2.rs

```diff
@@ -412,6 +412,8 @@ async fn reconnect_loop(
     let backoff = (|| {
         fail_point!("pd_client_v2_backoff", |s| {
             use std::str::FromStr;
+
+            use tikv_util::config::ReadableDuration;
```

Comment:
```suggestion
            use std::str::FromStr;
            use tikv_util::config::ReadableDuration;
```

Reply:
This is the "format" suggestion 😂 I don't want this either (